src/ManagementRoomOutput.ts

```diff
@@ -94,6 +95,10 @@ export default class ManagementRoomOutput {
      * @param isRecursive Whether logMessage is being called from logMessage.
      */
     public async logMessage(level: LogLevel, module: string, message: string | any, additionalRoomIds: string[] | string | null = null, isRecursive = false): Promise<any> {
+        if (level === LogLevel.ERROR) {
+            // Propagate any error to Sentry, for sorting and monitoring.
+            Sentry.captureMessage(`${module}: ${message}`, 'error');
```

Comment:
So `logMessage` is complicated. Technically we have two ways of reporting erros in mjolnir. Either you use `LogService` or you use `logMessage`. The latter being the only one that propagates to both the management room and the system logs. It's not right to only propogate when `logMessage` is used. 

We need to think about what situations `logMessage` is being used to begin with. Seems like a lot of the time it is used for any information that needs to be sent back to the management room, and when it is used for errors it doesn't seem to be encouraging user friendly / errors. I don't know how we work in a way where we can map errors to user facing descriptions but i'm sure there is inspiration to take from.

We need to unify how we report errors and how certain error conditions are mapped to an error message back to the management room.

Reply:
We also need a way to finally suppress the garbage from matrix-bot-sdk that is using the `LogService` but that's out of scope here

Reply:
> We need to unify how we report errors and how certain error conditions are mapped to an error message back to the management room.

Definitely. Can we do it in a followup?

Reply:
Probably yes, as it needs to be considered with how we report known errors back to the user in a friendlier way than "An error occurred see console log for details" (Something mjolnir actually says sometimes)